Doc/library/pathlib.rst

```diff
@@ -1055,3 +1071,42 @@ call fails (for example because the path doesn't exist):
       'Text file contents'
 
    .. versionadded:: 3.5
+
+Correspondence to tools in the :mod:`os` package
+------------------------------------------------
+
+If you're more familiar with :mod:`os.path` module, here's a correspondence
+table on how the same things may be accomplished with :mod:`pathlib`.
+
+.. note::
+
+   Although :func:`os.path.relpath` and :meth:`PurePath.relative_to` have some
+   overlapping use cases, :meth:`PurePath.relative_to` is more oriented towards
+   answering the question "Relative to a given parent directory, how do I reach
+   this descendant path?" whereas :func:`os.path.relpath` is implemented to
+   answer the "How do I get to path A from path B?".
+
+   If the parameter passed to :meth:`PurePath.relative_to` is not a parent
+   node of the object, :exc:`ValueError` exception is raised.
+
+============================   ==============================
+os and os.path                 pathlib
+============================   ==============================
+:func:`os.path.abspath`        :func:`Path.absolute`
+:func:`os.getcwd`              :func:`Path.cwd`
+:func:`os.path.abspath`        :meth:`Path.absolute`
+:func:`os.path.exists`         :meth:`Path.exists`
+:func:`os.path.expanduser`     :meth:`Path.expanduser` and
+                               :meth:`Path.home`
+:func:`os.stat`                :meth:`Path.group`
+:func:`os.path.isdir`          :meth:`Path.is_dir`
+:func:`os.path.isfile`         :meth:`Path.is_file`
+:func:`os.path.islink`         :meth:`Path.is_symlink`
+:func:`os.stat`                :meth:`Path.owner`
+:func:`os.stat`                :meth:`Path.stat`
```

Comment:
I would combine the three `os.stat` lines into one and simply comma-separate the pathlib equivalents. Otherwise someone looking for an `os.stat` equivalent will find the first solution and then never see the other two.